src/uwtools/tests/drivers/test_jedi.py

```diff
@@ -70,14 +71,52 @@ def cycle():
 
 @fixture
 def driverobj(config, cycle):
-    return jedi.JEDI(config=config, cycle=cycle, batch=True)
+    return JEDI(config=config, cycle=cycle, batch=True)
 
 
 # Tests
 
 
-def test_JEDI(driverobj):
-    assert isinstance(driverobj, jedi.JEDI)
+def test_JEDI():
+    for method in [
+        "_driver_config",
+        "_resources",
+        "_run_via_batch_submission",
+        "_run_via_local_execution",
+        "_runscript",
+        "_runscript_done_file",
+        "_runscript_path",
+        "_scheduler",
+        "_validate",
+        "_write_runscript",
+        "run",
+        "runscript",
+    ]:
+        assert getattr(JEDI, method) is getattr(Driver, method)
```

Comment:
Now that I've flown past a few of these, it makes me realize that perhaps we will never update these lists when/if we add new methods to the drivers. Can there be a test that will remind us to do that? I don't even know what that would look like.

Reply:
An automated version of this manually curated list would involve _generating_ this list by seeing which methods in a derived class are synonymous with the same-named method in the base class, but that would make this test a tautology.

So, I think that we either need to manually define this list to state what we believe to be true, or possibly just remove this test (and the corresponding tests in other modules) on the grounds that it doesn't provide much value, but...

What this test provides is the following: If today e.g `JEDI` inherits a method from `Driver` -- which has its own unit test(s) -- and tomorrow we implement a custom override method in `JEDI`, this test will fail, alerting us to the fact that we need to write a new unit test for that new custom override method.

I'm not sure I answered your question, though.